packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -76,14 +76,17 @@ import SharesNavigation from './SharesNavigation.vue'
 import SidebarToggle from './SidebarToggle.vue'
 import ViewOptions from './ViewOptions.vue'
 import { ViewMode } from 'web-pkg/src/ui/types'
+import * as ODSComponents from '@ownclouders/design-system/src/components'
 
 export default defineComponent({
   components: {
     BatchActions,
     ContextActions,
     SharesNavigation,
     SidebarToggle,
-    ViewOptions
+    ViewOptions,
+    // Remove when mixins are replaced
+    ...ODSComponents
```

Comment:
hack: we should be able to get rid of this with action composables: https://github.com/owncloud/web/pull/8542